examples/files.js

```diff
@@ -42,6 +42,7 @@ var files = {
 		"webgl_geometry_teapot",
 		"webgl_geometry_terrain",
 		"webgl_geometry_terrain_fog",
+		"webgl_geometry_terrain_fog_types",
```

Comment:
The terrain generation is not considered an important aspect of this demo, but it demonstrates fog well. Should the example be renamed to emphasize fog more?

Reply:
Not an important comment. Resolving to reduce noise.